blueprint/index.md

```diff
@@ -0,0 +1,197 @@
+
+## Data Table
+
+### Create a Data Table
+1. Go to **Admin**>**Architect**>**Data Table**
+2. First you will want to create a data table. Example name can be “Blacklist”. 
+2. You will then primarily need to have the Reference Key set as "ani". 
+3. Click "Save"
+
+   ![create a data table](images/datatable.gif "create a data table")
+
+### Add your Blacklist Numbers (to block incoming calls/queues)
+1. Open your Data Table
+2. Press "+" in the top right corner of the screen. 
+3. Under the ani header column, you will store all the phone numbers you wish to block from incoming calls or queues.
+4. Click "Save" 
+
+   NOTE: The phone numbers need to be formatted using e.164 without the "+", for example; 17705551234. This also allows for CSV import without many formatting issues. More information on this below. 
+
+   ![End-to-end user experience](images/add-number-to-blacklist.gif "End-to-end user experience")
+
+## Data Action
+
+You will need to create a Genesys Cloud data action that will be used for disconnecting interactions. This can be called “Disconnect interaction”. To set up the data action, make sure you already have an active Genesys Cloud data actions integration setup and follow the steps below or import the action in your ORG.
+
+### Create an OAuth client for use with a Genesys Cloud data action integration
+
+To enable a Genesys Cloud data action to make public API requests on behalf of your Genesys Cloud organization, use an OAuth client to configure authentication with Genesys Cloud.
+
+Create an OAuth client to use with the data action integration with a custom role.
+
+To create an OAuth Client in Genesys Cloud:
+
+1. Navigate to **Admin** > **Integrations** > **OAuth** and click **Add Client**.
+
+2. Enter the name (Example: Disconnect Interaction) for the OAuth client and select **Client Credentials** as the grant type. Click the **Roles** tab and assign the required role for the OAuth client.
+
+3. Click **Save**. Copy the client ID and the client secret values for later use.
+
+   ![End-to-end user experience](images/OAuth.gif "End-to-end user experience")
+
+   **Note:** Ensure that you **copy the client ID and client secret values** for each of the OAuth clients.
+
+### Add Genesys Cloud data action integration
+
+Add a Genesys cloud data action integration for each OAuth client being used with this blueprint to call the Genesys Cloud public API to:
+* Terminate the call of an inbound blacklisted caller
+
+To create a data action integration in Genesys Cloud:
+
+1. Navigate to **Admin** > **Integrations** > **Integrations** and install the **Genesys Cloud Data Actions** integration. For more information, see [About the data actions integrations](https://help.mypurecloud.com/?p=209478 "Opens the About the data actions integrations article") in the Genesys Cloud Resource Center.
+
+2. Enter a name for the Genesys Cloud data action, such as Update Genesys Cloud User Presence in this blueprint solution.
```

Comment:
Change "such as Update Genesys Cloud User Presence" to "Disconnect Interaction" to match your gif